back/app/policies/basket_policy.rb

```diff
@@ -2,20 +2,14 @@
 
 class BasketPolicy < ApplicationPolicy
   def create?
-    (
-      user&.active? &&
+    user&.active? &&
       (record.user_id == user.id) &&
       ProjectPolicy.new(user, record.participation_context.project).show? &&
       check_voting_allowed(record, user)
-    ) || (
-      user&.active? && UserRoleService.new.can_moderate?(record.participation_context, user)
```

Comment:
Why are moderators no longer able to do anything on baskets? Is that a security change so they can't unsubmit them?

Reply:
Moderators can still manage their own baskets, but they cannot see the content of other users baskets, submit a basket as someone else or edit someone else's basket.

Reply:
I feel like that would be very undemocratic. I discussed this during one of the standups we had.

Reply:
That's fine. I just wanted to be sure that this was an active decision. As you say, it does make sense